build.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN mkdir -p /usr/local/bin \
 RUN curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | bash -s -- -b $GOPATH/bin v1.45.0
 
 # Download misspell binary to bin folder in $GOPATH
-RUN curl -sfL https://raw.githubusercontent.com/client9/misspell/master/install-misspell.sh | bash -s -- -b $GOPATH/bin v0.3.4
+RUN go install github.com/client9/misspell/cmd/misspell@v0.3.4
```

Comment:
This never went through on my machine, this change fixed it. 

I had some troubles running integration tests such as: `(runtime: failed to create new OS thread (have 2 already; errno=22))` as I use a arm64 machine.